wp-content/plugins/tw-custom-tags/tw-custom-tags.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/*
+Plugin Name: TW Custom Tags
+Description: Creates the custom taxonomies for metadata
+*/
+
+require_once('taxonomies/taxonomy-city.php');
```

Comment:
@rpeterman-gp I've changed this file to break it out into separate files. Let me know if there are further enhancements I can make.